src/scrconf/dns_named.scr

```diff
@@ -22,7 +22,7 @@
 	    "comments"	: [ "^[ \t]*#.*$", "^[ \t]*$" ],
 	    "params"    : [
 		$[
-		    "match" : [ "([^ \t]+)[ \t]([^ \t]+(.*[^ \t]+)+)[ \t]*;$", "%s %s;" ],
+		    "match" : [ "([^ \t]+)[ \t]([^ \t]+(.*[^ \t]+)+)[ \t]*;", "%s %s;" ],
```

Comment:
My first try was
[ \t]*;[ \t]*$
which also did the job. But then I found this the even easier solution which the same outcome.

Honestly I am not sure of named.conf would even allow to have all the config in just one line without line breaks, but since our parser was not able to handle this even before this is at least no regression.

It is a L3 and so the priority should be a quick solution. The even better solution might be rewriting this using libaugeas – especially due to this comment in the code (by @jsrain):
// TODO: INI agent isn't optimal, in this file multiple occurences of one key
//       in one section may be possible, section on one line is handled as a
//       single value, and many many other problems are to be expected....
//       should be replaced in future by some better agent
But this might better be a separate PBI.